sql/src/main/java/com/vmware/ddlog/util/sql/CalciteToH2.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Is this copied from somewhere?
If not, please use the same license as the rest of the code.
And the year is wrong too.